tools/InstallGit.ps1

```diff
@@ -0,0 +1 @@
+& "C:\Program Files\PowerShell\7\pwsh.exe" -c "Invoke-WebRequest -Uri https://github.com/git-for-windows/git/releases/download/v2.28.0.windows.1/Git-2.28.0-64-bit.exe -OutFile .\git.exe; .\git.exe /VERYSILENT /NORESTART"
```

Comment:
This is great! 
An improvement to consider for the future is to get the latest.